app/views/Project/Tagging/Sources/SourcesTable/index.tsx

```diff
@@ -47,18 +46,20 @@ const leadsKeySelector: (d: Lead) => number = (d) => d.id;
 
 const statusIconMap: Record<Lead['status'], ReactNode> = {
     pending: <VscLoading />,
-    validated: <IoCheckmarkCircleOutline />,
-    processed: null,
+    validated: undefined,
+    processed: <VscLoading />,
 };
 const statusVariantMap: Record<Lead['status'], 'gradient2' | 'accent' | 'complement1'> = {
     pending: 'gradient2',
-    validated: 'accent',
-    processed: 'complement1',
+    processed: 'gradient2',
+    validated: 'complement1',
 };
+
+// NOTE: This will be removed after introduction of graphQL
 const statusLabelMap: Record<Lead['status'], string> = {
     pending: 'In Progress',
-    validated: 'Validated',
-    processed: 'Tagged',
+    processed: 'In Progress',
```

Comment:
Are you sure both of these mean 'in progress'?

Reply:
This will be fixed after Sameer's PR.
Right now this is done to mimic future changes to tags.